### DISCUSSION TITLE
Visualize most used tags of a book as a word cloud

### DISCUSSION THREAD

**Discussion by tobias-klein** [ezra-bible-app/ezra-bible-app#450] [Ideas]
I am thinking about adding a feature for visualizing the most used tags of a book right on top of it (before book introduction) as a word cloud. This would be very easy to implement with a library like this one:
http://mistic100.github.io/jQCloud/demo.html

Any feedback before I move forward with this?

**Comment by zhuiks** [comment]
I think it's a great idea. Maybe show the cloud only if you have more than 5 or 10 tags in the book...
I would avoid using jQuery for new features, though.

**Reply by tobias-klein** [reply]
I can check if there is a good vanilla js alternative, didn't find one when I looked lastly (there are various react js libs, but don't want that either)
